src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        String[] mashine = new String[] {"Truck", "Bulldozer", "Excavator"};
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
```

Comment:
Good job using abstract references for `Machine` types. This allows for polymorphism in the code.